systemjs/systemjs.d.ts

```diff
@@ -8,6 +8,7 @@ interface System {
   defined: any;
   amdDefine: () => void;
   amdRequire: () => void;
+  normalize(name: string, parentName: string): string;
```

Comment:
I think normalize has a 3 arguments.  `name, referrerName, referrerAddress`. not 2.